greedy_methods/interval_scheduling.py

```diff
@@ -0,0 +1,27 @@
+# Interval Scheduling: https://en.wikipedia.org/wiki/Interval_scheduling
+
+
+def interval_scheduling(intervals):
```

Comment:
Please add doctests

Reply:
sure I will make the changes and psh my PR again